configs/FLYCOLORF7_AIO_HD/config.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        FLYCOLORF7_AIO_HD
+#define MANUFACTURER_ID   FLCO
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6500
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6500
+#define USE_MAX7456
+
+#define BEEPER_PIN PC13
+#define MOTOR1_PIN PC6
+#define MOTOR2_PIN PC7
+#define MOTOR3_PIN PB0
+#define MOTOR4_PIN PB1
+#define MOTOR5_PIN PB6
+#define MOTOR6_PIN PB8
+#define MOTOR7_PIN PB7
+#define MOTOR8_PIN PC9
+#define RX_PPM_PIN PA15
+#define LED_STRIP_PIN PA8
+#define UART1_TX_PIN PA9
+#define UART2_TX_PIN PA2
+#define UART3_TX_PIN PC10
+#define UART4_TX_PIN PA0
+#define UART5_TX_PIN PC12
+#define UART1_RX_PIN PA10
+#define UART2_RX_PIN PA3
+#define UART3_RX_PIN PC11
+#define UART4_RX_PIN PA1
+#define UART5_RX_PIN PD2
+#define I2C2_SCL_PIN PB10
+#define I2C2_SDA_PIN PB11
+#define LED0_PIN PB2
+#define SPI1_SCK_PIN PA5
+#define SPI2_SCK_PIN PB13
+#define SPI3_SCK_PIN PB3
+#define SPI1_SDI_PIN PA6
+#define SPI2_SDI_PIN PB14
+#define SPI3_SDI_PIN PB4
+#define SPI1_SDO_PIN PA7
+#define SPI2_SDO_PIN PB15
+#define SPI3_SDO_PIN PB5
+#define CAMERA_CONTROL_PIN PB9
+#define ADC_VBAT_PIN PC1
+#define ADC_RSSI_PIN PC3
+#define ADC_CURR_PIN PC2
+#define BARO_CS_PIN C15
+#define FLASH_CS_PIN C8
+#define MAX7456_SPI_CS_PIN B12
+#define GYRO_1_EXTI_PIN C4
+#define GYRO_1_CS_PIN A4
+#define PINIO1_PIN C14
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP(  0, PC6,  1, 0) \
+    TIMER_PIN_MAP(  1, PC7,  1, 0) \
+    TIMER_PIN_MAP(  2, PB0,  3, 0) \
+    TIMER_PIN_MAP(  3, PB1,  2, 0) \
+    TIMER_PIN_MAP(  4, PB6,  1, 0) \
+    TIMER_PIN_MAP(  5, PB8,  1, 0) \
+    TIMER_PIN_MAP(  6, PA15, 1, 0) \
+    TIMER_PIN_MAP(  7, PB9,  2, 0) \
+    TIMER_PIN_MAP(  8, PA8,  1, 0) \
+    TIMER_PIN_MAP(  9, PB7,  1, 0) \
+    TIMER_PIN_MAP( 10, PC9,  2, 0)
+
+#define ADC1_DMA_OPT 1
+
+#define DEFAULT_FEATURES        (FEATURE_OSD | FEATURE_LED_STRIP)
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
+
+#define SERIALRX_UART SERIAL_PORT_USART1
+
+#define DEFAULT_PID_PROCESS_DENOM 2
+#define SERIALRX_PROVIDER SBUS
```

Comment:
Does this need to be defined?

Reply:
Agree this should be cloud build defined (except for onboard control links using SPI).